Framework/Nexus/src/NexusFile.cpp

```diff
@@ -195,53 +195,48 @@ void File::initOpenFile(std::string const &filename, NXaccess const am) {
     msg << "File::initOpenFile(" << filename << ", " << am << ") failed";
     throw NXEXCEPTION(msg.str());
   } else {
-    m_fid = temp_fid;
+    m_pfile = std::make_shared<H5::H5File>(temp_fid);
```

Comment:
Should this be using move semanatics? 
```suggestion
    m_pfile = std::make_shared<H5::H5File>(std::move(temp_fid));
```

Reply:
Here `temp_fid` is a primitive (`int64_t`).  The most efficient way to move it _is_ copying, so `std::move()` isn't necessary.  If it were some other object, then this would be the correct place for `std::move()`